posthog/api/organization.py

```diff
@@ -16,24 +17,25 @@ class PremiumMultiorganizationPermissions(permissions.BasePermission):
 
     def has_permission(self, request: request.Request, view) -> bool:
         if (
+            # make multiple orgs only premium on self-hosted, since enforcement of this is not possible on Cloud
             not getattr(settings, "MULTI_TENANCY", False)
             and request.method in CREATE_METHODS
-            and not request.user.organization.is_feature_available("organizations_projects")
+            and (
+                request.user.organization is None
+                or not request.user.organization.is_feature_available("organizations_projects")
```

Comment:
Note (in general): the naming of this is sort of bad, because it really looks like a feature flag call though it isn't.

Reply:
Hm, I see, you can make an issue about this if you want.